acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -6,5 +6,111 @@ def ruby_command(host)
       end
       module_function :ruby_command
     end
-  end
-end
+
+    module CAUtils
+      def initialize_ssl
+        hostname = on(master, 'facter hostname').stdout.strip
+        fqdn = on(master, 'facter fqdn').stdout.strip
+
+        if master.use_service_scripts?
```

Comment:
:+1: to making the code unconditional.

The term "use service scripts" refers to whether there are service scripts for start/stopping the puppet server, and is implemented in beaker: https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L176. Based on the return value, beaker will start/stop the puppet master differently, e.g. https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/helpers/puppet_helpers.rb#L165, such as if you're using `with_puppet_running_on`. But yeah, I wouldn't worry about that.